tools/istio-iptables/pkg/capture/run.go

```diff
@@ -359,6 +359,10 @@ func (cfg *IptablesConfigurator) Run() {
 	cfg.iptables.AppendRule(iptableslog.UndefinedCommand, constants.ISTIOINBOUND, constants.NAT, "-p", constants.TCP, "--dport",
 		cfg.cfg.InboundTunnelPort, "-j", constants.RETURN)
 
+	// Create a rule for invalid drop
```

Comment:
I think this rule is not self explanatory, and could use some extensive comments on why we do this. I actually don't understand it myself. Ideally linking to some docs recommending we do this

Reply:
Actually after reading the k8s blog it explains it pretty well, could link there in the docs.

Reply:
Thanks @howardjohn, here is the doc link  https://kubernetes.io/blog/2019/03/29/kube-proxy-subtleties-debugging-an-intermittent-connection-reset/

Reply:
@howardjohn  I added more details for the debugging. In our case we are still using the default value 0 nf_conntrack_tcp_be_liberal, so reset the Out-of-Order (OFO) packet is the expected behavior and we have similar concern to enable it at node level and we don't know why kernel use 0 as default value. 
So config the DROP rule in app pod network namespace would be lower risk than config it in node level.